docs/config/cluster.rst

```diff
@@ -179,6 +179,18 @@ Collecting stats
 
     Using a very small value can cause a high load on the cluster.
 
+.. _stats.service.max_bytes_per_sec:
+
+**stats.service.max_bytes_per_sec**
+  | *Default:*    ``0``
+  | *Runtime:*   ``yes``
```

Comment:
Given that part of the motivation for this change is that users experience unexpected performance hiccups due to the analyze I think we should enable the throttling by default.

Not sure what a good value would be, probably worth running some tests on a lower specced cluster and aim for a value that errors on the safe-but-maybe-slow side of things?

Reply:
Enabling by default sounds good to me.

On choosing default - maybe we can align with existing` *_bytes_per_sec` settings - would make it `40mb` which looks like a reasonable lower bound (just randomly looked up one of the I/O throttling  `_bytes_per_sec` settings and found a [table](https://www.elastic.co/guide/en/elasticsearch/reference/master/recovery.html#recovery-settings)).

Reply:
40 sounds good, but some manual testing to see the impact would be nice anyways.

Reply:
Hi @mfussenegger, I changed default to 40mb and checked on dev cluster:
https://tools.cr8.net/grafana/d/hkhCfqdmk/dev-cluster?var-node=All&from=now-3h&to=now 


I run ANALYZE 4 times with the following values:` 0, 40mb, 40mb, 0` - latency got higher with enabled throttling as expected.

<img width="1772" alt="Screenshot 2022-09-28 at 14 08 41" src="https://user-images.githubusercontent.com/7446940/192775046-2f4181e4-2dc8-4316-835d-e18d1a942039.png">

Reply:
Chart for `node_disk_read_bytes_total`, run with 0, 20, 60 - the higher value I set for throttling, the lower IOPS gets.

<img width="344" alt="Screenshot 2022-09-28 at 16 30 37" src="https://user-images.githubusercontent.com/7446940/192806757-9fbc1678-000d-4712-8e61-70ce43b26d41.png">